federation-js/src/composition/validate/preComposition/__tests__/duplicateEnumValue.test.ts

```diff
@@ -66,6 +68,12 @@ describe('duplicateEnumValue', () => {
       Array [
         Object {
           "code": "DUPLICATE_ENUM_VALUE",
+          "locations": Array [
```

Comment:
Repeat: can we capture all locations of the duped enum values?

Reply:
^ same as above